sermons.php

```diff
@@ -60,13 +60,18 @@ class SermonManager {
 	 */
 	private static $instance = null;
 
+	/**
+	 * @var string The domain(sic) name to use when scoping variables and keywords (since WP hasn't heard of namespacing)
+	 */
+	const PLUGIN_DOMAIN = 'sermon-manager-for-wordpress';
+
 	/**
 	 * Construct.
 	 */
 	public function __construct() {
 		// Define constants (PATH and URL are with a trailing slash).
 		define( 'SM_PLUGIN_FILE', __FILE__ );
-		define( 'SM_PATH', dirname( SM_PLUGIN_FILE ) . '/' );
+		define( 'SM_PATH', dirname( SM_PLUGIN_FILE ) . DIRECTORY_SEPARATOR );

```

Comment:
Nice!

Reply:
This was part of 7c18a14d361ece39507d77de5fed9fa90878d424 that I reverted. I can put it back in if you like..

Reply:
However you wish. We didn't have any issues with `'/'` so far